clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleRegionalServerGroupCachingAgent.groovy

```diff
@@ -491,9 +496,15 @@ class GoogleRegionalServerGroupCachingAgent extends AbstractGoogleCachingAgent i
       def instanceMetadata = instanceTemplate?.properties?.metadata
       if (instanceMetadata) {
         def metadataMap = Utils.buildMapFromMetadata(instanceMetadata)
-        def loadBalancerNameList = metadataMap?.get(LOAD_BALANCER_NAMES)?.split(",")
-        if (loadBalancerNameList) {
-          serverGroup.asg.loadBalancerNames = loadBalancerNameList
+        def regionalLBNameList = metadataMap?.get(LOAD_BALANCER_NAMES)?.split(",")
```

Comment:
Same thing here (about sharing the common code between the two server group caching agents).